examples/06-patchman.yaml

```diff
@@ -6,13 +6,17 @@
 ## - filter       Only perform updates for machines whose names match
 ##                the filter
 ## - autoremove   Set to non-empty value to also autoremove old packages
+## - withnewpkgs  Set to non-empty value to also install new packages
 
 ## Usage 1 (to autoremove packages):
 ##  $ amaltheia -s this-file.yaml -v autoremove="1"
 
 ## Usage 2 (without autoremove)
 ##  $ amaltheia -s this-file.yaml -v filter="FILTER" autoremove=false
 
+## Usage 3 (to install new packages):
+##  $ amaltheia -s this-file.yaml -v filter="FILTER" withnewpkgs="1"
```

Comment:
`withnewpkgs=true`, since this is a boolean variable.

Reply:
done